editor/src/clj/editor/defold_project.clj

```diff
@@ -656,6 +656,17 @@
                 :command :build}
                {:label "Rebuild"
                 :command :rebuild}
+               {:label "Instance Count On Build"
+                :children (mapv (fn [i]
+                                  {:label (str i (if (> i 1)
+                                                   " Intances"
+                                                   " Instance"))
```

Comment:
is it good enough? Should we change manu names somehow here?
![CleanShot 2024-08-02 at 11 43 01@2x](https://github.com/user-attachments/assets/edf29953-bddc-4885-bdb7-28aa9888d00b)

Reply:
Fixed 
![Uploading CleanShot 2024-08-05 at 13.18.34@2x.png…]()